src/aleph/http/server.clj

```diff
@@ -107,9 +107,7 @@
   (log/error e "error in HTTP handler")
   {:status 500
    :headers {"content-type" "text/plain"}
-   :body (let [w (java.io.StringWriter.)]
-           (.printStackTrace e (java.io.PrintWriter. w))
-           (str w))})
+   :body "Internal Server Error"})
```

Comment:
It was pretty weird to return the details of the server internal error to the client (at least for security reasons)

Reply:
Good call. We may want to enable this with a configurable flag (if you trust your network), but it definitely shouldn't be the default.

Reply:
Yes, but let's not add this flag is not requested, right?

Reply:
Yeah, probably no need for that yet.